lib/dictionary.ml

```diff
@@ -0,0 +1,24 @@
+type dictionary = (string, string) Hashtbl.t
```

Comment:
you can change the type of this to `(string, unit) Hashtbl.t` instead; since you just want to check whether words are present in the hash table and you don't need to retrieve any data associated with them